rpa-mapper/src/main/java/uk/gov/hmcts/cmc/rpa/mapper/SealedClaimJsonMapper.java

```diff
@@ -63,7 +63,7 @@ private JsonArray mapDefendants(List<TheirDetails> defendants) {
                 .add("address", mapAddress(defendant.getAddress()))
                 .add("correspondenceAddress", defendant.getServiceAddress().map(this::mapAddress).orElse(null))
                 .add("emailAddress", defendant.getEmail().orElse(null))
-                .add("businessName", extractOptionalFromSubclass(defendant, SoleTraderDetails.class, SoleTraderDetails::getBusinessName))
+                .add("businessName", addPrefixToBusinessName(extractOptionalFromSubclass(defendant, SoleTraderDetails.class, SoleTraderDetails::getBusinessName)))
```

Comment:
extractOptionalFromSubclass is a misleading name - It returns string. Also too much was happening inline already. Rest are actioned.

Reply:
Will look into the name to make it better.

Your solution did not contribute towards reducing number of things that are happening inline :-P

Please do not use extra function that operates on the result of extractOptionalFromSubclass - we have optionals for a reason, use it instead of working with nulls. There are nicer ways of solving the problem.